dev/com.ibm.ws.concurrent.persistent_fat_multiple/build.gradle

```diff
@@ -10,10 +10,20 @@
  *******************************************************************************/
 //TODO this can be moved to the wlp-gradle/subprojects/fat.gradle 
 
-def isOracleAvailable = file("../prereq.dbtest/lib/Oracle/jars/Oracle/18.3.0.0.0/ojdbc8_g.jar").exists()
-
 configurations {
-  jdbcDrivers
+  //Transitive dependencies in Oracle Maven pom are not necessary when just using
```

Comment:
instead of manually excluding each transitive dependency, you can disable transitive dependencies for a configuration like this:
```groovy
configurations {
  jdbcDrivers {
    transitive = false
  }
}
```